variation/vrs_representation.py

```diff
@@ -135,21 +146,36 @@ def vrs_allele(self, ac: str, interval: models.SequenceInterval,
             return None
 
         seq_id, w = self.seqrepo_access.translate_identifier(
-            allele.location.sequence_id._value, "ga4gh")
+            allele.location.sequence_id._value, "ga4gh"
+        )
         if seq_id:
             seq_id = seq_id[0]
             allele.location.sequence_id = seq_id
             allele.location._id = ga4gh_identify(allele.location)
             allele._id = ga4gh_identify(allele)
-            return allele.as_dict()
+            allele_dict = allele.as_dict()
+            try:
+                Allele(**allele_dict)
+            except ValidationError as e:
+                errors.append(str(e))
+                return None
+            else:
+                return allele_dict
         else:
             errors.append(w)
             return None
 
     def to_vrs_allele(
-            self, ac: str, start: int, end: int, coordinate: str,
-            alt_type: AltType, errors: List, cds_start: int = None,
-            alt: str = None) -> Optional[Dict]:
+        self,
+        ac: str,
+        start: int,
+        end: int,
+        coordinate: str,
```

Comment:
this might be a good place for an enum?

Reply:
I didn't update the type hint, we do use enums for coordinates